spark/v3.1/build.gradle

```diff
@@ -216,6 +216,7 @@ project(':iceberg-spark:iceberg-spark-runtime-3.1_2.12') {
     implementation (project(':iceberg-snowflake')) {
       exclude group: 'net.snowflake' , module: 'snowflake-jdbc'
     }
+    implementation project(':iceberg-gcp')
```

Comment:
I'd remove this. We can iterate much more quickly if we get this code into Iceberg. This addition causes us to need to validate the size of the iceberg-gcp module as well as the transitive dependencies it pulls in. That will block this PR until we do the analysis. I think we should focus on getting the logic of the catalog right, and then add it for distribution.

Reply:
Sounds good. I removed the changes to spark build files.